deploy/examples/create-external-cluster-resources.py

```diff
@@ -757,6 +729,7 @@ def _convert_hostname_to_ip(self, host_name, port, ip_type):
             host_ip_type = self._invalid_endpoint(host_addr + ":80")
             import socket
 
+            ip = []
```

Comment:
Here I would prefer to raise an error in an else statement because if the ip type is not IPv6 or IPv4 the error is not nice in the return statement. That should also take care of the pylint error

Reply:
we get this from ceph cmd `orch host ls`, so we expect it to be a valid ip